src/rpc/net.cpp

```diff
@@ -178,6 +178,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
             obj.pushKV("banscore", statestats.nMisbehavior);
             obj.pushKV("synced_headers", statestats.nSyncHeight);
             obj.pushKV("synced_blocks", statestats.nCommonHeight);
+            obj.pushKV("tx_inflight", statestats.nTxInFlight);
+            obj.pushKV("tx_process", statestats.nTxProcess);
```

Comment:
do these need to be added to the help text?

Reply:
are they even worth keeping? i mostly added them for debugging, and i am using them in one of the test cases, but they're pretty obscure info.

Reply:
added help text anyway